### DISCUSSION TITLE
Add support for `size` validation rule

### DISCUSSION THREAD

**Discussion by manuel-watchenterprise** [dedoc/scramble#591] [Ideas]
`min` and `max` are supported already, however, [size](https://laravel.com/docs/11.x/validation#rule-size) is not yet. It's a nice to have feature. This would be actually adding both `min` and `max` with the argument, I guess.

```php
[
  'options' => ['bail', 'required', 'array', 'min:2', 'max:2'],
  // equivalent
  'options' => ['bail', 'required', 'array', 'size:2'],
]
```

**Comment by romalytvynenko** [comment]
@manuel-watchenterprise released in `0.11.25` https://github.com/dedoc/scramble/releases/tag/v0.11.25
